Source/UI/NewUI/BillingForm/Default Implementation/BillingForm/Header/Title/DefaultHeaderLabelFormStyle.swift

```diff
@@ -1,8 +1,21 @@
 import UIKit
 
-struct DefaultHeaderLabelFormStyle: CKOLabelStyle {
-    var isHidden: Bool { false }
-    var text: String { "billingAddressTitle".localized(forClass: CheckoutTheme.self) }
-    var font: UIFont { UIFont(graphikStyle: .medium, size: 24) }
-    var textColor: UIColor { .codGray }
+struct DefaultHeaderLabelFormStyle: CKOElementStyle {
+    var backgroundColor: UIColor
+    var isHidden: Bool
+    var text: String
+    var font: UIFont
+    var textColor: UIColor
+
+    init(backgroundColor: UIColor = .clear,
+         isHidden: Bool = false,
+         text: String = "billingAddressTitle".localized(forClass: CheckoutTheme.self),
+         font: UIFont = UIFont(graphikStyle: .medium, size: 24),
+         textColor: UIColor = .codGray ) {
```

Comment:
Just realised that these initialisers on the style objects are all internal. Is there any reason we don't want to just set those properties as default values and drop the initialiser?

ie:
```
struct DefaultHeaderLabelFormStyle: CKOElementStyle {
     var backgroundColor: UIColor = .clear
     var isHidden: Bool = false
     var text: String = "billingAddressTitle".localized(forClass: CheckoutTheme.self)
     var font: UIFont = UIFont(graphikStyle: .medium, size: 24)
     var textColor: UIColor = .codGray
}
```